services/modrinth/modrinth-game-versions.service.js

```diff
@@ -0,0 +1,41 @@
+import { BaseModrinthService, documentation } from './modrinth-base.js'
+
+export default class ModrinthGameVersions extends BaseModrinthService {
+  static category = 'platform-support'
+
+  static route = {
+    base: 'modrinth/game-versions',
+    pattern: ':projectId',
+  }
+
+  static examples = [
+    {
+      title: 'Modrinth Game Versions',
+      namedParams: { projectId: 'AANobbMI' },
+      staticPreview: this.render({ versions: ['1.19.2', '1.19.1', '1.19'] }),
+      documentation,
+    },
+  ]
+
+  static defaultBadgeData = { label: 'game versions' }
+
+  static render({ versions }) {
+    if (versions.length) {
+      return {
+        message: versions.join(' | '),
+        color: 'blue',
+      }
+    } else {
+      return {
+        message: 'missing',
+        color: 'red',
+      }
+    }
```

Comment:
Instead of manually returning an 'error' response here, lets require that the `game_versions` array has at least 1 element in the Joi schema using

```js
Joi.array().items(Joi.string()).min(1).required() // note min(1)
```

If we do this, an error will be thrown at the point we validate the response and we can just drop the length check here.